### DISCUSSION TITLE
Guidance on using Badge component with "Beta" in the left hand navigation?

### DISCUSSION THREAD

**Discussion by mjanssen2021** [twilio-labs/paste#2142] [Q&A] [answered]
<img width="546" alt="Screen Shot 2022-01-18 at 11 19 27 AM" src="https://user-images.githubusercontent.com/78432794/149986583-a02c6005-d769-48a8-82a0-706e061506d3.png">

We want to label our new feature as a "beta" since this will be going into private beta first before being generally released to the public. Can we or should we use both a badge component and the "beta" navigation indicator for this? Or should we use one over the other? Is there a preference from the paste team on this?

**Comment by jweintraub** [comment, marked as answer]
Hi Margot,
You can go ahead and use the badge next to the H1 as you have it here. The nav will continue to show beta as it does today. I'm going to add this guidance to our console documentation.
thanks

**Reply by kdocherty6** [reply]
Does the beta tag go on the H1 for every page of the product or only the main starting page?

**Reply by serifluous** [reply]
cc @kendal-kirby for the most current console guidance.

**Reply by kendal-kirby** [reply]
I'm currently looking into this to see if there have been already established guidelines from Jo or Console team. If not, Kaitlyn and I will work together make a call and document the decision.

**Reply by kendal-kirby** [reply]
Beta tags only need to be on the main page for the feature. [Guidelines here](https://internal-product-docs.twilio.com/docs/console/console-product-design/UX%20Guidelines/?q=console+guidelines#beta-tags)

**Comment by jweintraub** [comment]
Margot - one more thing, please make sure that that badge next to the H1 is clickable and opens the Beta modal like it does from the nav. Thanks!
